app/hub/Views/HomeView/HomeView.scss

```diff
@@ -73,14 +73,25 @@
 	padding: 0;
 	background-color: #979797;
 }
-.HomeView__plus {
+.HomeView__upgradeContainer {
+	display: flex;
 	margin-top: 22px;
 	padding: 22px 12px;
 	background-color: #f0f3f4;
 	border-radius: 4px;
+
+	&.purple-border {
+		box-shadow: 0 0 8px 5px #920094;
+		border: solid 1.6px #920094;
+	}
 }
-.HomeView__plusFeature {
-	padding: 0 36px;
+.HomeView__upgradeIcon {
+	background-image: upgradeShieldIcon(#DAF4FF, #4a4a4a);
+	margin: 0 42px;
+	height: 50px;
+	width: 50px;
+	background-repeat: no-repeat;
+	background-position: center center;
```

Comment:
Looks like you can cut `background-position` too.

Reply:
Cut!